### PR TITLE
chore: Let dependabot update GitHub actions

### PR DESCRIPTION
**Pull request by reneleonhardt**
Let bots do some of the tedious work 😄

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```
Comment (dvarrazzo) [inline .github/dependabot.yml:3]: Which dependencies would this end up updating?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (dvarrazzo) [inline .github/dependabot.yml:10]: Please add a newline at the end of the file.

File: .github/dependabot.yml (hunk 1 above)
Comment (reneleonhardt) [reply .github/dependabot.yml:3]: As far as I can see only doc/requirements.txt:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#pip-and-pip-compile
But the PRs will not be auto-merged, you still have to merge the suggestions yourself, but you don't have to update manually anymore 😅

Comment (dvarrazzo) [reply .github/dependabot.yml:3]: Yes, I'm not a heavy user of dependabot, but many projects I maintain still receive security-oriented upgrade suggestions.

However with the docs is tricky: we don't have automatic tests that verify that the documentation is rendered correctly. Merging the change would generate new docs, which might be broken. So I don't think I want to run the risk.

Can you please limit the updates to the github actions, and set it to a monthly update? I don't think a weekly update is requested.

Comment (reneleonhardt) [reply .github/dependabot.yml:3]: Sure thing.

**Pull request merged**